tests/main/views/test_brief_response.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
do we need this line https://stackoverflow.com/a/58652741/238230
seems to suggest you don't need it in Python 3?

Reply:
You're right, we don't need it for our Python 3 apps (my IDE is still set up to complain about Python 2 violations). Will remove.